content/tutorials/presence.textile

```diff
@@ -137,110 +206,145 @@ blang[ruby].
     end
   ```
 
-  "See this step in Github":https://github.com/ably/tutorials/commit/presence-ruby-step3
+You are now in the presence set. The name you will appear as is the @client ID@ that you used to instantiate the Ably client library in step 2.
 
-We're now in the presence set. The name we'll appear as is the @client ID@ that we included in the Realtime constructor in step 2.
+Similarly, you can call @channel.presence.leave@ to leave the presence set.
 
-Similarly, we can call @channel.presence.leave@ to leave the presence set.
-
-(Side note: while in this tutorial we had free choice of @client ID@, in most situations you don't want that. Instead you want your server to constrain what @client ID@ people can use -- for example, to their logged-in username -- so that no-one can pretend to be someone else. When using "Token authentication":/core-features/authentication#token-authentication, your server can issue tokens with a specific @client ID@. When such a token is used by someone, their connection is bound to that @client ID@.)
+(Side note: while in this tutorial you can choose any @client ID@, in most situations you don't want that. Instead you want your server to constrain what @client ID@ people can use -- for example, to their logged-in username -- so that no-one can pretend to be someone else. When using "Token authentication":/core-features/authentication#token-authentication, your server can issue tokens with a specific @client ID@. When such a token is used by someone, their connection is bound to that @client ID@.)
```

Comment:
Just personal preference but the entire paragraph being in brackets feels off.

Reply:
Done! 👍